packages/auth/src/graphql/schema.graphql

```diff
@@ -45,12 +45,17 @@ type GrantEdge {
 
 input GrantFilter {
   identifier: FilterString
+  state: FilterGrantState
 }
 
 input FilterString {
   in: [String!]!
 }
 
+input FilterGrantState {
+  notIn: [GrantState!]!
+}
```

Comment:
Why `notIn` instead of `in`?

Reply:
This endpoint now returns revoked grants, when previously it did not. So the thinking with `notIn` was giving the client a way to return the same set of results as before.

Reply:
i added and `in` filter for good measure